environment.yml

```diff
@@ -38,5 +38,6 @@ dependencies:
   - flake8
   - pip:
     - dcps
+    - thorlabs-apt-device
```

Comment:
You will need to update this to the new package you decided to use. Make sure to check whether it can be installed conda, in which case you can delete this line here and add your package further above to the list of packages that get automatically installed from conda. If it can only do pip, then you replace it here.

Reply:
I just checked on the GutHub page of the new package and they distribute it neither through conda nor pip, so you'll have to add a GitHub installation like like for example for Vimba. This is not ideal but should do it for now.

The more reason for @johanmazoyer to open that issue I mentioned so that we keep track of alternatives and investigate them later.

Reply:
I think it exists on pip ?  https://pypi.org/project/thorlabs-apt/

Reply:
Oh thank goodness. It's not listed on the GitHub page so I haven't seen it.